transport-classes-kqueue/src/main/java/io/netty/channel/kqueue/AbstractKQueueChannel.java

```diff
@@ -363,7 +364,8 @@ private void evSet0(short filter, short flags, int fflags) {
         }
     }
 
-    abstract class AbstractKQueueUnsafe extends AbstractUnsafe {
+    @UnstableApi
+    public abstract class AbstractKQueueUnsafe extends AbstractUnsafe {
```

Comment:
how is this exposed as public ?

Reply:
It is exposed as a return type from the `protected` method `newUnsafe` in `AbstractKQueueStreamChannel`, which is `public` and not `final`.